tests/Spec.hs

```diff
@@ -804,6 +807,23 @@ main = hspec $ do
       fromJust (Project.lookupPullRequest prId state') `shouldSatisfy`
         (\pr -> Project.approval pr == Just (Approval (Username "deckard") Project.Merge 0))
 
+    it "test tag" $ do
```

Comment:
Thanks for adding a test! I assume this is testing the merge on friday from the exempted bot. Could you rename `test tag` to describe that specific test?